x-pack/metricbeat/module/aws/vpn/_meta/fields.yml

```diff
@@ -4,3 +4,15 @@
     `vpn` contains the metrics from Cloudwatch to track usage of VPN related resources.
   release: beta
   fields:
+    - name: metrics
+      type: group
+      fields:
+        - name: TunnelState.avg
+          type: integer
```

Comment:
Since this is `avg` shouldn't be `double`?

Reply:
Yeah, good catch!